### PR TITLE
AngularSize now defined in lucuma-catalog, instead of being in target model.

### PR DESCRIPTION
**Pull request by rpiaggio**
This PR goes on top of https://github.com/gemini-hlsw/explore/pull/1446

### CODE REVIEW

File: explore/src/main/scala/explore/targets/TargetSearchResult.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import japgolly.scalajs.react.Reusability
+import lucuma.catalog.AngularSize
+import lucuma.catalog.CatalogTargetResult
+import lucuma.core.model.Target
+
+final case class TargetSearchResult(
+  targetWithOptId: TargetWithOptId,
+  angularSize:     Option[AngularSize]
+) {
+  lazy val optId: Option[Target.Id] = targetWithOptId.optId
```
Comment (cquiroz) [inline explore/src/main/scala/explore/targets/TargetSearchResult.scala:18]: Why lazy?

**Review by cquiroz — APPROVED** [review]
Looks good, I guess we should look into storing the angular size on the user preferences or is it going to be on the odb?

File: explore/src/main/scala/explore/targets/TargetSearchResult.scala (hunk 1 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/targets/TargetSearchResult.scala:18]: So as not to allocate it unless it's needed.

**Comment by rpiaggio** [comment]
> Looks good, I guess we should look into storing the angular size on the user preferences or is it going to be on the odb?

This PR is to actually remove it from the ODB. I agree we should initialize visualization preferences with this and store in the users preferences DB. I'll create a story for this.

**Pull request merged**